main/src/ca/uwaterloo/flix/language/ast/ReducedAst.scala

```diff
@@ -129,14 +129,20 @@ object ReducedAst {
 
   case class HandlerRule(op: OpSymUse, fparams: List[FormalParam], exp: Expr)
 
+  sealed trait ExpParam {
```

Comment:
Could this just be Param? Are there any non-exam params?

Reply:
I wanted to mirror TypeParam. but maybe Param is not too confusing no